lib/wasix/src/runners/wasi.rs

```diff
@@ -353,6 +353,10 @@ impl crate::runners::Runner for WasiRunner {
             }
         }
 
+        if let Some(cwd) = cmd.metadata().annotation::<String>("cwd")? {
```

Comment:
Shouldn't this also be a `wasi` annotation?
Not every kind of runner would definitely have a CWD, and it fits together with the other wasi settings.